packages/blade/package.json

```diff
@@ -80,7 +86,8 @@
     "types:copy-declarations:native": "copyfiles -u 1 \"src/**/*.d.ts\" build/types/native",
     "build:generate-root-imports": "rollup -c && node ./scripts/generateRootImports.js",
     "build:clean-declarations": "rm -rf build/types",
-    "build:react": "FRAMEWORK=REACT rollup -c && node ./scripts/generateRootImports.js",
+    "build:react-dev": "FRAMEWORK=REACT NODE_ENV=development rollup -c && node ./scripts/generateRootImports.js",
```

Comment:
@snitin315 we shall remove `node ./scripts/generateRootImports.js` from all the places. there's already `build:generate-root-imports` which does this job so this is redundant and has no effect but increases the build time. Its not related to your changes but some place this got missed and added to package.json in the past I feel, can you remove it now?

Reply:
@kamleshchandnani Sure, I'll remove it.